versions.gradle

```diff
@@ -1,7 +1,7 @@
 ext {
     kotlin_version = '1.3.72'
-    okhttp_version = '4.7.2'
-    okhttp_url_version = '4.7.2'
+    okhttp_version = '3.12.12'
+    okhttp_url_version = '3.12.12'
```

Comment:
Any reason why downgrading this two dependencies is necessary?

Reply:
sdk compatibility with older than 21, since okhttp version 4 set minimum sdk to 21.
you can read more about okhttp [here](https://square.github.io/okhttp/)

Reply:
Then in any case this library should increase its min sdk to 21 since it has a dependency on a third party library which requires skd 21, or at least make the okhttp module an exception. I mean, there's a very good reason why a very well known and used library like okhttp increased its min sdk to 21...

Reply:
https://github.com/tonyofrancis/Fetch/blob/fb88cb83227d3a057b2e2c0bab959941ec674e1a/versions.gradle#L11
Since this project started supporting sdk lower than 21 and still shows min sdk 14, Then why not continue do so unless you want the maintainer to maintain two separate branches and that is a lot of work, Okhttp in the other hand have a lot of maintainers and that why they can maintain multiple branches.

Also okhttp backport critical fixes to v3 branch, And the main reason they increased its min sdk is because "These platforms lack support for TLS 1.2".

Plus upgrading okhttp to v4 is it seems a mistake from the maintainer, Otherwise he would change the entire project min sdk to 21 not just upgrading okhttp to v4